From 2cc03e84e5a12d3f19ab83aa5efa00ab0db45b5c Mon Sep 17 00:00:00 2001
From: Rander Wang <rander.wang@linux.intel.com>
Date: Sun, 29 Sep 2019 09:29:59 +0800
Subject: [PATCH] FROMGIT: ALSA: sof: disable debug message for production release

It is clear for release version without many debug messages.

Signed-off-by: Rander Wang <rander.wang@linux.intel.com>
---
 sound/soc/sof/Makefile        | 2 --
 sound/soc/sof/intel/Makefile  | 2 --
 sound/soc/sof/xtensa/Makefile | 2 --
 3 files changed, 6 deletions(-)

diff --git a/sound/soc/sof/Makefile b/sound/soc/sof/Makefile
index 33c3ded2b7e2..8f14c9d2950b 100644
--- a/sound/soc/sof/Makefile
+++ b/sound/soc/sof/Makefile
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
-ccflags-y += -DDEBUG
-
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 		control.o trace.o utils.o
 
diff --git a/sound/soc/sof/intel/Makefile b/sound/soc/sof/intel/Makefile
index 03b22924d0c4..84a65c80f62a 100644
--- a/sound/soc/sof/intel/Makefile
+++ b/sound/soc/sof/intel/Makefile
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
-ccflags-y += -DDEBUG
-
 snd-sof-intel-byt-objs := byt.o
 snd-sof-intel-hsw-objs := hsw.o
 snd-sof-intel-bdw-objs := bdw.o
diff --git a/sound/soc/sof/xtensa/Makefile b/sound/soc/sof/xtensa/Makefile
index 34b27be0f7be..cc89c7472a38 100644
--- a/sound/soc/sof/xtensa/Makefile
+++ b/sound/soc/sof/xtensa/Makefile
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
-ccflags-y += -DDEBUG
-
 snd-sof-xtensa-dsp-objs := core.o
 
 obj-$(CONFIG_SND_SOC_SOF_XTENSA) += snd-sof-xtensa-dsp.o
-- 
2.17.1

